CHANGELOG.rst

```diff
@@ -18,6 +18,13 @@ Unreleased
   deprecated.  See https://github.com/openedx/edx-sphinx-theme/issues/184 for
   more details.
 
+[7.1.0] - 2023-04-24
```

Comment:
One last change, now that it's ready to go.
```suggestion
[7.1.0] - 2023-05-03
```

Reply:
I'll go ahead and merge, and will update this in a separate PR. (I don't have permissions to update this branch.)

Reply:
@timmc-edx Thanks!